translations/it.php

```diff
@@ -536,6 +570,7 @@
 $_MODULE['<{ps_checkout}prestashop>translations_f367b9c00a9870296e60afbd3bd2a52a'] = 'Integrazione quasi terminata!';
 $_MODULE['<{ps_checkout}prestashop>translations_f40be703dee22a1ce61f7b75190687b3'] = 'Inviando il presente modulo acconsenti alla possibile raccolta da parte di PrestaShop S.A. dei dati da te forniti per la creazione del tuo account PrestaShop Checkout. Creando un account riceverai annunci commerciali da PrestaShop';
 $_MODULE['<{ps_checkout}prestashop>translations_f55d7f720fdc5e3ff693e4b82d4b6a72'] = 'transazione/i';
```

Comment:
```suggestion
$_MODULE['<{ps_checkout}prestashop>translations_f55d7f720fdc5e3ff693e4b82d4b6a72'] = 'transazione';
```

Reply:
Done

Reply:
Actually, it's plural form for "transaction" in IT : transazioni
There are many places where it's translated like it, we should keep like it.